ecosystem/sf-indexer/aptos-sf-indexer/src/counters.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
can also add a `gauge` I believe, for latest version per processor :-)

Reply:
What's a gauge?

Reply:
from metrics, it'd either be `IntCounter` or `IntGauge`

Reply:
cool cool. I'll do block instead.